pkg/project/registry/projectrequest/delegated/delegated.go

```diff
@@ -155,9 +168,37 @@ func (r *REST) Create(ctx kapi.Context, obj runtime.Object) (runtime.Object, err
 		return nil, kapierror.NewInternalError(err)
 	}
 
+	// if there's no rolebinding in the project, go ahead and return
+	if lastRoleBinding == nil {
+		return r.openshiftClient.Projects().Get(projectName)
+	}
+
+	r.waitForRoleBinding(projectName, lastRoleBinding.Name)
 	return r.openshiftClient.Projects().Get(projectName)
 }
 
+func (r *REST) waitForRoleBinding(namespace, name string) {
+	// we have a rolebinding, the we check the cache we have to see if its been updated with this rolebinding
+	// if you share a cache with our authorizer (you should), then this will let you know when the authorizer is ready.
+	// doesn't matter if this failed.  When the call returns, return.  If we have access great.  If not, oh well.
+	err := wait.PollImmediate(50*time.Millisecond, 30*time.Second, func() (bool, error) {
```

Comment:
Use wait.ExponentialBackoff and try to go for 5-6 iterations. (10ms, 100ms, 1s, 5s, 10s, 30s)